### ISSUE TITLE
portaudio.def for msvc is missing an entry for Pa_GetVersionInfo()

### ISSUE THREAD

**Issue by closetcodebrews** [PortAudio/portaudio#322]
(Please use the mailing list for support requests and general discussion. This is only for actual bugs.)

**Describe the bug**
The portaudio.def file included as a part of the msvc build is missing an entry for Pa_GetVersionInfo.
Came across this as I started try / use your examples/pa_devs.c file.  It tries to call the above method but fails.  You end up with an error 

Error   LNK2019 unresolved external symbol Pa_GetVersionInfo referenced in function main  

**To Reproduce**
1.  follow instructions to build portaudio on windows.  compiled 64 bit dll.
2.  create a new windows console application.  compile and make sure you get the default "hello world" message.
3.  copy in contents from pa_devs.c example file into the main cpp file of your console app. 
4.  change project properties to include the lib and headers. 
5.  copied the 64bit dll into the console application folder to simplify things. 
6.  Try to build console application. 

**Expected behavior**
System will return version information on the console.

**Actual behavior**
Error   LNK2019 unresolved external symbol Pa_GetVersionInfo referenced in function main  

**Desktop (please complete the following information):**
 - OS: Windows
 - OS Version 10
 - PortAudio version:  grabbed master branch just 2 days ago.
 - If Windows or Linux, which Host API (e.g. WASAPI):

**Additional context**

Patch was simple.  
Just added Pa_GetVersionInfo @69 into that file and it built just fine.
console application started working and listed all devices on my PC.

**Comment by RossBencina** [comment]
We currently have two .def files, one in build/msvc and one in cmake_support. please use the same @ number in both. I recommend using @35, the lowest available number after Pa_Sleep. Insert the entry after Pa_Sleep, not later, where the host api extension symbols are defined.

**Comment by RossBencina** [comment]
Related #323

**Comment by RossBencina** [comment]
Fixed by https://github.com/PortAudio/portaudio/pull/574. Not sure why this ticket wasn't closed already. I have reconfirmed that `Pa_GetVersionInfo` is present in both .def files and that the ordinal @35 in both files matches. There is a comparison of all ordinals in both files here (they are perfectly matching):

https://docs.google.com/spreadsheets/d/1-J8BDLMq4SlUpskLLNh3YVDX4Xu2ry7OCXXS-l5bWXo/edit?usp=sharing
